knowledge_repo/app/templates/markdown-base.html

```diff
@@ -15,6 +15,11 @@
             <button type="button" class="btn btn-default btn-raw">
               View Raw Markdown
             </button>
+            {% if is_author and webeditor_buttons %}
```

Comment:
Simplified to move the boolean logic into the python file. This whole file could use some of that simplification.